src/scripts/main.js

```diff
@@ -1 +1,76 @@
 'use strict';
+
+document.querySelector('#form').addEventListener('submit', (e) =>
+  e.preventDefault()
+);
+
+document.querySelector('#button').addEventListener('click', function(event) {
+  event.preventDefault();
+});
+
+document.addEventListener('DOMContentLoaded', function() {
+  document.querySelector('.header__icon:last-child')
+    .addEventListener('click', openNav);
+
+  document.querySelector('.header__logo--black')
+    .addEventListener('click', closeNav);
+
+  document.querySelector('.closebtn').addEventListener('click', closeNav);
+
+  const anchorLinks = document.querySelectorAll('a[href^="#"]');
+
+  anchorLinks.forEach(function(link) {
+    link.addEventListener('click', function(event) {
+      event.preventDefault();
+
+      const target = document.querySelector(this.getAttribute('href'));
+
+      target.scrollIntoView({
+        behavior: 'smooth',
+      });
+    });
+  });
```

Comment:
You can achieve this without JS. Because now the page does not work as intended.